go/libraries/doltcore/sqle/sqlselect_test.go

```diff
@@ -969,216 +975,6 @@ var AsOfTests = []SelectTest{
 	},
 }
 
-// SQL is supposed to be case insensitive. These are tests of that promise.
-// Many of these are currently broken in go-myqsl-server. The queries return the correct results in all cases, but the
-// column names in the result schemas often have the wrong case. They sometimes use the case from the table, rather
-// than the case of the expression in the query (the correct behavior). This is a minor issue, but we should fix it
-// eventually.
-var CaseSensitivityTests = []SelectTest{
```

Comment:
You think this functionality is well represented in existing engine tests?

Reply:
I do, and most of these tests were skipped anyway